conda/cli/main_clean.py

```diff
@@ -16,25 +17,57 @@
 _EXTS = (*CONDA_PACKAGE_EXTENSIONS, *(f"{e}.part" for e in CONDA_PACKAGE_EXTENSIONS))
 
 
-def find_tarballs():
+def _getsize(*parts: str, warnings: Optional[List[Tuple[str, Exception]]] = None) -> int:
```

Comment:
```suggestion
def _get_size(*parts: str, warnings: Optional[List[Tuple[str, Exception]]] = None) -> int:
```
This is definitely more of a nit, feel free to ignore.